src/test/java/teammates/test/cases/util/FieldValidatorTest.java

```diff
@@ -367,9 +368,15 @@ public void invalidityInfoFor_validGender_returnEmptyString() {
     public void invalidityInfoFor_invalidGender_returnErrorString() {
```

Comment:
Also need to test `nationality` since you edit the `nationality` things in `FieldValidator`?